content/en/docs/architecture/release-gating.md

```diff
@@ -67,88 +36,215 @@ Good candidates include:
 * Projects or internal tools that depend on OpenShift APIs heavily
 * Community projects being evaluated for inclusion into OpenShift or the portfolio
 
-Poor candidates include:
-
-* Jobs with low signal-to-noise ratio (low pass rates)
-* Random software from the internet (that may contain vulnerabilities that could be used to attack our CI
-  infrastructure)
-* Personal projects that use similar APIs and patterns to projects already being tested
+For layered products that extend the control plane of the cluster, they must be able
+to tie into OpenShift's CI and block releases. For example, OCS is a part of the OpenShift platform,
+and if an upgrade breaks OCS, we have broken part of our platform. Extending our testing to
+these components is a requirement to perform regression and early verification.
 
 {{< alert title="Note" color="info" >}}
 OpenShift CI creates thousands of clusters a week - we can certainly afford to run a few more tests. However, we want to
 spread that investment to derive the maximum benefit for everyone, so having OpenShift CI run 30 variations of your
 exact same tests in slightly different configurations 4 times a day may not be the best way to test.
 {{< /alert >}}
 
-### An Introduction to Aggregated Jobs
+### Pre-merge testing
+
+Detecting a regression before it ends up in the release payload at all is the ultimate 
+form of release testing. It is the most expensive form because each 
+change is tested in isolation. Additionally, due to the scope of OpenShift and it's 
+configurations, it is not possible to test everything against every code change.
+
+Repository owners should make an educated selection of the most impactful jobs that 
+have a high likelihood of detecting a problem. As the team learns, this list should 
+be modified. For example, if a change to the repository causes an incident (see below),
+the job that was affected should become a presubmit job on that repository.
+
+We generally recommend you have coverage for both the parallel conformance and
+serial suites, as well as an upgrade job.
+
+Typically, jobs which are present on many repositories that span multiple teams
+should also be payload blocking.
+
+### Payload blocking
```

Comment:
encouraging deeper analysis of informing jobs is something we should explicitly encourage.